dev-environment.yml

```diff
@@ -33,5 +33,6 @@ dependencies:
   - sphinx-gallery
 
   - pip:
-    - git+https://github.com/GlacioHack/GeoUtils.git
+    - geoutils==0.0.7
```

Comment:
Should it not be moved out of the pip commands, directly in the conda install?

Reply:
Conda takes a day to update, pip is faster